dali/operators/math/normalize/normalize.cc

```diff
@@ -22,27 +22,29 @@
 namespace dali {
 
 DALI_SCHEMA(Normalize)
-  .DocStr(R"(Normalizes the input by removing mean and dividing by standard deviation.
+  .DocStr(R"code(Normalizes the input by removing the mean and dividing by the standard deviation.
 
-The mean and standard deviation can be calculated internally for specified subset of axes or
-can be externally provided as `mean` and `stddev` arguments.
+The mean and standard deviation can be calculated internally for the specified subset
+of axes or can be externally provided as the ``mean`` and ``stddev`` arguments.
 
 The normalization is done following the formula::
 
-    out = scale * (in - mean) / stddev + shift
+  out = scale * (in - mean) / stddev + shift
 
-The expression assumes that *out*, *in* are equally shaped tensors whereas *mean* and *stddev*
-may be either tensors of same shape or scalars or a mix of these. The expression follows
-*numpy* broadcasting rules.
+The formula assumes that *out* and *in* are equally shaped tensors, but *mean* and *stddev* might
+be either tensors of same shape, scalars, or a mix of these.
 
-Sizes of (non-scalar) `mean` and `stddev` must have either extent of 1, if given axis is reduced,
-or match the corresponding extent of the input. A dimension is considered reduced if it's listed
-in `axes` or `axis_names`. If neither `axes` nor `axis_names` argument is
-present, the set of reduced axes is inferred by comparing input shape to the shape of
-mean/stddev arguments, but it is enforced that the set of reduced axes is the same for all tensors
-in the batch.
+.. note::
+  The expression follows the *numpy* broadcasting rules.
 
-Examples of valid argument combinations:
+Sizes of the non-scalar ``mean`` and ``stddev`` must have an extent of 1, if the specified axis
+is reduced, or match the corresponding extent of the input. A dimension is considered reduced
+if it is listed in ``axes`` or ``axis_names``. If neither the ``axes`` nor the ``axis_names``
+argument is present, the set of reduced ``axes`` is inferred by comparing the input
```

Comment:
```suggestion
argument is present, the set of reduced axes is inferred by comparing the input
```
just axes, not the argument.

Reply:
Done